views/overlays/umsk/game16by9.hbs

```diff
@@ -25,6 +25,28 @@
         document.getElementById('rplayer').innerText = data.rplayer || "";
         document.getElementById('lscore').innerText = data.lscore || "";
         document.getElementById('rscore').innerText = data.rscore || "";
+
+        if(data.lCharacter) {
+          var leftIcon = document.createElement('img');
+          leftIcon.id = 'leftCharacter';
+          leftIcon.className = data.lCharacter;
+          leftIcon.style.zoom = '200%';
+          //src is required to remove the dumb border
+          leftIcon.src = 'http://upload.wikimedia.org/wikipedia/commons/c/ce/Transparent.gif';
+          document.getElementById('lplayer').appendChild(leftIcon);
```

Comment:
Could these images be pre-created in the HTML, and then the javascript just changes which class is applied? I think that would cut down on the code needed here and also move the hardcoded image URL out of the JS. Also, do you have any opinions on making it a `<div>` rather than an `<img>`? I'm not sure what standard practice is for that